zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1387,6 +1387,17 @@ the variable does.
     problems will arise. This is really a sanity check. ZooKeeper is
     designed to store data on the order of kilobytes in size.
 
+* *jute.maxbuffer.extrasize*:
+    (Java system property: **zookeeper.jute.maxbuffer.extrasize**)
+    **New in 3.6.0:**
```

Comment:
You have committed **New in 3.5.7** for branch-3.5.

I feel we should stick to 3.5.7, though I don't know which ever we will release first

Reply:
What should be the version in branch-3.4? 3.4.15 or 3.5.7?

Reply:
I think we won't port it to 3.4, it is a maintenance only version

Reply:
OK